libusb/hotplug.c

```diff
@@ -215,6 +215,7 @@ void usbi_hotplug_exit(struct libusb_context *ctx)
 	}
 
 	/* free all discovered devices. due to parent references loop until no devices are freed. */
```

Comment:
Not directly affecting your PR, but I notice this comment seems outdated (already since 87a8578). It used to be a do-while loop.

Reply:
I simplified the comment